pkgs/applications/networking/firewalk/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib
+, stdenv
+, fetchurl
+, libnet
+, libpcap
+, libdnet
+}:
+
+stdenv.mkDerivation rec {
+    pname = "firewalk";
+    version = "master";
+
+    src = fetchurl {
+        url = "https://salsa.debian.org/pkg-security-team/firewalk/-/archive/debian/master/firewalk-debian-master.tar.gz";
+        hash = "sha256-d45Hm8WFTiVLPY2fPwnLdyapXLnBQaEJvzIgMdlzl08=";
+    };
+
+    buildInputs = [ libnet libpcap libdnet ];
+
+    meta = with lib; {
+        description = "A gateway ACL scanner";
+        homepage = "https://salsa.debian.org/pkg-security-team/firewalk";
+        licence = [  ];
+        maintainers = with maintainers; [  ];
```

Comment:
I feel like leaving these out, unless they contain meaning values, is a better option here.